docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -510,6 +510,60 @@ The default is 30303.
 !!!note
     This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#exposing-ports). 
 
+### privacy-enabled
+
+```bash tab="Syntax"
+--privacy-enabled=<true|false>
```

Comment:
value is optionnal and without value, will swithc the boolean. So as default is false, if passed `--privacy-enabled` without value it will make it true. Then the following would be better IMO:
```suggestion
--privacy-enabled[=<true|false>]
```